tools/integration_tests/kernel_list_cache/infinite_kernel_list_cache_test.go

```diff
@@ -526,8 +526,9 @@ func TestInfiniteKernelListCacheTest(t *testing.T) {
 	}
 
 	// Define flag set to run the tests.
+	// TODO: Replace metadata-cache-ttl-secs with something better
```

Comment:
Should we write high-level why are we disabling the metadata cache for this testsuite?

Reply:
Added